contrib/centos/aziot-identity-service.spec

```diff
@@ -78,29 +78,29 @@ make -j \
 # For each of CS, IS, KS: create group, create user, create home directory (in case user already exists from a previous install
 # but the user deleted the directory manually)
 
+if ! %{_bindir}/getent group aziotks >/dev/null; then
+    %{_sbindir}/groupadd -r aziotks
+fi
+if ! %{_bindir}/getent passwd aziotks >/dev/null; then
+    %{_sbindir}/useradd -r -g aziotks -c 'aziot-keyd user' -s /sbin/nologin -d /var/lib/aziot/keyd aziotks
+fi
+
 if ! %{_bindir}/getent group aziotcs >/dev/null; then
     %{_sbindir}/groupadd -r aziotcs
 fi
 if ! %{_bindir}/getent passwd aziotcs >/dev/null; then
     %{_sbindir}/useradd -r -g aziotcs -c 'aziot-certd user' -s /sbin/nologin -d /var/lib/aziot/certd aziotcs
+    %{_sbindir}/usermod -aG aziotks aziotcs
 fi
-mkdir -p /var/lib/aziot/certd
```

Comment:
Please double check this line can be removed. According to the RPM packaging guide, `%dir` only owns the folder but folder still needs to be created. Thanks.

Reply:
I confirmed installing the package does create the homedirs.